libr/core/cmd_seek.c

```diff
@@ -347,9 +347,58 @@ static int cmd_seek(void *data, const char *input) {
 		}
 		r_core_seek_base (core, input);
 		break;
+	case 'j':
+		{
+			RList /*<uintptr_t>*/ *addrs = r_list_new ();
+			RList /*<char *>*/ *names = r_list_newf (free);
+			RList *list = r_io_sundo_list (core->io, '!');
+			ut64 lsz = 0;
+			ut64 i;
+			RListIter *iter;
+			RIOUndos *undo;
+			if (list) {
+				r_list_foreach (list, iter, undo) {
+					char *name = NULL;
+
+					core->flags->space_strict = true;
+					RFlagItem *f = r_flag_get_at (core->flags, undo->off, true);
+					core->flags->space_strict = false;
+					if (f) {
+						if (f->offset != undo->off) {
+							name = r_str_newf ("%s + %d\n", f->name,
+									(int)(undo->off- f->offset));
+						} else {
+							name = strdup (f->name);
+						}
+					}
+					if (!name) {
```

Comment:
nope, it doesn't, this is the case where it's no flags here, so we make an empty.